browser/ui/webui/brave_webui_source.cc

```diff
@@ -6,8 +6,12 @@
 
 #include <map>
 
+#if !defined(OS_ANDROID)
 #include "brave/components/brave_rewards/resources/grit/brave_rewards_resources.h"
 #include "components/grit/brave_components_resources.h"
```

Comment:
these I know you won't be including now, but brave_rewards_resources.h should be good